config/compose/cluster.yml

```diff
@@ -0,0 +1,39 @@
+version: "2"
+services:
+  jenkins-master-service:
+    image: ubuntucore/snappy-jenkins-master
+    ports:
+      - 8080:8080
+    volumes:
+      - ${JENKINS_HOME}:/var/jenkins_home:rw
+    restart: always
+    user: root
+
+  snappy-proxy:
+    depends_on:
+      - jenkins-master-service
+    image: ubuntucore/snappy-jenkins-proxy
+    ports:
+      - 8081:80
+    restart: always
+
+  jenkins-slave-vivid-1:
+    image: ubuntucore/snappy-jenkins-slave-vivid
+    command: -username admin -password snappy -executors 2 -name jenkins-slave-vivid-1 -labels vivid -master http://jenkins-master-service:8080
+    volumes:
+      - ${JENKINS_HOME}:/var/jenkins_home
+    privileged: true
+    restart: always
+
+  jenkins-slave-xenial-1:
+    extends: jenkins-slave-vivid-1
+    image: ubuntucore/snappy-jenkins-slave-xenial
+    command: -username admin -password snappy -executors 2 -name jenkins-slave-xenial-1 -labels xenial -master http://jenkins-master-service:8080
+
+  jenkins-slave-xenial-2:
+    extends: jenkins-slave-xenial-1
+    command: -username admin -password snappy -executors 2 -name jenkins-slave-xenial-2 -labels xenial -master http://jenkins-master-service:8080
+
+  jenkins-slave-xenial-3:
+    extends: jenkins-slave-xenial-1
+    command: -username admin -password snappy -executors 2 -name jenkins-slave-xenial-3 -labels xenial -master http://jenkins-master-service:8080
```

Comment:
What's this -username? I thought that was going to be the user that runs the jenkins jobs.
We have a problem because currently the jobs are being run as root. Looking at the docs I thought I could use -username jenkins-slave, but now I'm confused.